pkg/dogstatsd/server.go

```diff
@@ -782,6 +822,13 @@ func (s *Server) storeMetricStats(sample metrics.MetricSample) {
 	ms.Tags = strings.Join(s.debugTagsAccumulator.Get(), " ") // we don't want/need to share the underlying array
 	s.Debug.Stats[key] = ms
 
+	name := fmt.Sprintf("Metric Name: %s |", ms.Name)
+	tags := fmt.Sprintf(" Tags: {%s} |", ms.Tags)
+	count := fmt.Sprintf(" Count: %d |", ms.Count)
+	ls := fmt.Sprintln(" Last Seen : ", ms.LastSeen)
```

Comment:
Would it make sense to use a human-readable representation of the timestamp instead of the raw view?

Reply:
I updated the format to more like this:
```2023-02-13 14:13:21 | [DOGSTATSD_STATS] | [INF] Metric Name: dd_mac_book_count_metric.count | Tags: {environment:prod  dogstatsd_metric:true  secondary_tag:new_metric?} | Count: 1 | Last Seen: 2023-02-13 14:13:03.657791 -0500 EST m=+3.196348841```